llvm/test/Transforms/LoopVectorize/icmp-uniforms.ll

```diff
@@ -48,7 +49,7 @@ for.end:
 ; CHECK-NEXT: <x1> vector loop: {
 ; CHECK-NEXT: vector.body:
 ; CHECK-NEXT:   EMIT vp<[[CAN_IV:%.+]]> = CANONICAL-INDUCTION
-; CHECK-NEXT:   WIDEN-INDUCTION %iv = phi 0, %iv.next, ir<1>
+; CHECK-NEXT:   WIDEN-INDUCTION ir<%iv> = phi ir<0>, vp<[[NEXT_WIV:%.+]]>, ir<1>
```

Comment:
What is the meaning of `ir<1>`?
Is it necessary after this patch decompose `WidenIntOrFPInduction`?

Reply:
in dump `ir<>` represents VPValue that has underlying LLVM IR value.
Prior to that changeset `WidenIntOrFPInduction::print` method [printed LLVM IR PHINode](https://github.com/llvm/llvm-project/pull/82021/files/372daf08f62bcf602408e002e2b4c73e222aa9f9#diff-34abe4c3cd34aa7a9664bbd204834248455635ba80b8a9ba9506d8c3e6b94d95L1089). I've changed it to print VPValues used by the `WidenIntOrFPInduction`